ceph_deploy/hosts/centos/install.py

```diff
@@ -198,11 +200,8 @@ def repo_install(distro, reponame, baseurl, gpgkey, **kw):
 
     # set the right priority
     if kw.get('priority'):
-        distro.packager.install_priorities_plugin()
-        logger.warning(
-            'ensuring that {repo_path} contains a high priority'.format(
-                repo_path=repo_path)
-        )
+        if isinstance(distro.packager, Yum):
```

Comment:
I'm surprised - do you consider `isinstance` to be bad practice?  It seems like a very OO way of doing things.  We don't need to know anything about the implementation of the class (i.e. attributes like `name` or `executable`, just merely that the packager is a class doing Yum.

I also like that if I didn't care whether it was Yum or DNF, I could do `isinstance(distro.packager, RPMBaseManager)`.

What are the drawbacks of `isinstance` ?

> I don't think that a list is needed in Apt and could be normalized to apt-get and then this would work.

Whether to switch on `executable` aside, I'm happy to make `executable` be just a single executable name (`apt-get` for Apt) and add the common options another way.  I was just getting lazy. :)